docs/next/guides/integrate-with-angular/angular-modules.md

```diff
@@ -95,3 +97,15 @@ Register your modules, using the registering functions that you imported (for th
   })
   export class AppModule { }
   ```
+
+## Related articles
+
+#### Related guides
+
+- [Modules](@/guides/building-and-testing/modules.md)
+- [Performance](@/guides/advanced-topics/performance.md)
+
+#### Related blog articles
+
+- [Modularizing to improve the developer experience](https://handsontable.com/blog/modularizing-to-improve-the-developer-experience)
+- [Handsontable 11.0.0: modularization for React, Angular, and Vue](https://handsontable.com/blog/handsontable-11.0.0-modularization-for-react-angular-and-vue)
```

Comment:
To be removed

Reply:
Done: [9bcb64a](https://github.com/handsontable/handsontable/pull/9374/commits/9bcb64adf0edd372f9af4cd32cab05e56930d315)